platform-sdk/swirlds-common/src/main/java/com/swirlds/common/stream/internal/LinkedObjectStreamValidateUtils.java

```diff
@@ -225,4 +227,17 @@ public static <T extends SelfSerializable> Pair<StreamValidationResult, Hash> va
         }
         return Pair.of(StreamValidationResult.OK, endHash);
     }
+
+    // Code duplicated in order to avoid wasting time on the event stream which will be removed soon
+    public static boolean verifySignature(final byte[] data, final Signature signature, final PublicKey publicKey) {
+        try {
+            final java.security.Signature sig = java.security.Signature.getInstance(
+                    SignatureType.RSA.signingAlgorithm(), SignatureType.RSA.provider());
+            sig.initVerify(publicKey);
+            sig.update(data);
+            return sig.verify(signature.getSignatureBytes());
+        } catch (final Exception ignored) {
+        }
```

Comment:
This may be extremely difficult to debug. Please consider re-throwing a `RuntimeException` wrapping the original exception here, or at least log the original exception. Otherwise, please add comments in code explaining why any and all exceptions are safe to silently swallow here (there's hardly ever a good reason for this, unless it's a very specific and relatively benign exception such as the `InterruptedException` or similar.)

Reply:
The main reason I kind of glossed over this is that this code will be removed soon. I now added logging that was there in the original method that was removed.

Reply:
Thank you!